ci/create_experment.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+'''Basic python script to create an experment directory on the fly from a given
+
+yaml file for the arguments to the two scripts below in ${HOMEgfs}/workflow
+
+where ${HOMEgfs} is specified within the input yaml file.
+
+${HOMEgfs}/workflow/setup_expt.py
+${HOMEgfs}/workflow/setup_xml.py
+'''
+
+import os, sys, socket
+import glob
```

Comment:
I don't think you are using this anymore.
```suggestion
```

Reply:
oh right.